pkg/adapter/v2/cloudevents.go

```diff
@@ -106,6 +107,10 @@ func setTimeOut(duration time.Duration) http.Option {
 	}
 }
 
+func isRetriableFunc(sc int) bool {
+	return sc < 200 || sc >= 300
+}
+
```

Comment:
I'd actually like to see something more aligned w/ this:
https://github.com/knative/eventing/blob/main/pkg/kncloudevents/message_sender.go#L163-L166

Reply:
Thanks!